README.md

```
@@ -320,7 +320,42 @@ WAY4PAY_API_URL="<WAY4PAY_API_URL>"
+Start the **ASP.NET Core** application:
+```bash
+dotnet run --project VictoryCenter.WebAPI
+```
+
+Alternatively, if using **Visual Studio**:
+1. Open the **VictoryCenter.sln** solution.
+2. Set **VictoryCenter.WebAPI** as the startup project.
+3. Press **F5** to run in debug mode.
+
+Once started, the app will be accessible at:
+- **http://localhost:5000** (For HTTP)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)